iothub/device/src/ClientOptions.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.Devices.Client
     public class ClientOptions
     {
         /// <summary>
-        /// The Digital Twin Model Id associated with the device client instance.
+        /// The Digital Twins model Id associated with the device client instance.
```

Comment:
you can lower case digital twins too, per your other inquiry

Reply:
ok